### PR TITLE
Add firefoxvschrome.com

### PR DESCRIPTION
**Pull request by heubergen**

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -62,6 +62,7 @@ <h1>Hi I&rsquo;m Markus</h1>
         <dd>
           <a href="https://dnszensur.ch/">DNSzensur</a>,
           <a href="https://web.archive.org/web/20171022202658/https://ueberwacht.ch/">ueberwacht.ch</a>
+	  <a href="https://firefoxvschrome.com/">firefoxvschrome.com</a>
```
Comment (mritzmann) [inline www/index.html]: Can you please use spaces for indentations?

**Review by mritzmann — CHANGES_REQUESTED** [review]
Thanks for your contribution! Added a remark.

File: www/index.html (hunk 1 above)
Comment (heubergen) [reply www/index.html]: fixed

**Comment by mritzmann** [comment]
Decided not to add the project firefoxvschrome.com to my website.

**Pull request closed without merging**